src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -42,6 +42,7 @@ export const AddressInput = ({ rules, placeholder, enableQr = false }: AddressIn
         value={value}
         variant='filled'
         data-test='send-address-input'
+        data-1p-ignore
```

Comment:
gm @woodenfurniture
https://flaviocopes.com/how-to-disable-1password-in-an-input-field/

Reply:
🧠